pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -79,6 +84,12 @@ func TestConnectionStore_addAndUpdateConn(t *testing.T) {
 		TupleReply:      *revTuple2,
 		IsActive:        true,
 	}
+	tuple3, revTuple3 := makeTuple(&net.IP{10, 10, 10, 10}, &net.IP{20, 20, 20, 20}, 6, 5000, 80)
+	testFlow3 := flowexporter.Connection{
+		TupleOrig:  *tuple3,
+		TupleReply: *revTuple3,
```

Comment:
If you always end up dereferencing the tuples, I think it may make more sense to just return values (and not pointers) in `makeTuple`.

Reply:
Sure.. Any insight on the copy of return value from a function VS copy from dereferencing? I believe the first one is little efficient because of return registers.

Reply:
I think performance is not an issue here, since the struct is small, the function is not called many times, and this is test code. It's also not always faster to return a pointer. So I would favor readability. If you were embedding a point in `flowexporter.Connection`, I would recommend using a pointer instead.

You are also using the tuple as an immutable value in a way, so I think passing it around by value works well.